src/tests/class_tests/openms/source/FlagSet_test.cpp

```diff
@@ -87,6 +87,10 @@ START_SECTION(FlagSet())
 }
 END_SECTION
 
+START_SECTION(~FlagSet())
```

Comment:
Not_testable?

Reply:
I think in general, we just call the destructor in these tests which allows e.g., ASan to catch problems with missing virtual destructors etc.